src/style/internal/shared_input-select-picker.scss

```diff
@@ -2,34 +2,28 @@
 // components such as input fields, select dropdowns and pickers
 
 $mdc-textarea-background: $mdc-theme-background;
-// $mdc-textarea-disabled-background: $mdc-theme-background;
-
 $mdc-text-field-background: $mdc-theme-background;
```

Comment:
What do you mean? That we should move them into that file instead? 

I guess they are used in other files as well, like picker. But I don't remember.

Reply:
Can we check? I don't like when changes are made and there's cruft left all over the place. The code will become an unmaintainable quagmire in no time like that.

Reply:
Well… the idea of having this "shared" file was to make these available in all components that are based on the same core (inout fields). This is because we have these custom built components that look like native MD stuff.

At this moment it seems only one of the constants (used for background color of input fields) is used in two different places:
![image](https://user-images.githubusercontent.com/35954987/101776390-2360ae00-3af1-11eb-9e07-9105869131b2.png)
Rest are only used in `outlined-style-overrides.scss`. Nevertheless, if we move them there now, I can imagine a future that we may move them back here again for another component. (which can be OK too –except having to preform additional commits perhaps).

I don't see any logical reason for not leaving them in this shared file, given the intention behind it. It's a common place for potentially common things. But that (in my own rationale) shouldn't mean that all of those component that are pulling-in this file must exactly use everything in this file too; otherwise we must move them into individual files. For me, this is like the `mixin` file, and doesn't really make it unmaintainable.

Anyhow, no strong opinions about their placement. I'm fine either way.

Reply:
If they are still useful, we should of course keep them.

I was just wondering, because these variables are called `$mdc-text-field-background` and `$mdc-textarea-background`, so presumably they are meant to be used in for those specific components, with the possible exception of using them to make some other component look exactly like them… So if we overwrite this value for our text-fields, I couldn't really see where else we might want to use them without also using those new values? 🙂

Reply:
> If they are still useful, we should of course keep them.

This's hard to judge. If it feels better that they get moved there for now, let's do it. Do you want me to move them into separate files and do a fixup? As I said, for me both work equally alright really.

Reply:
If it's hard to judge now, it will be ever harder next time we come across this code, and don't know why those variables are there.

I've checked:

- We don't use `$mdc-textarea-background` nor `$mdc-text-field-background` anywhere in our code.
- The only place they are used in MDC is in `mdc-text-field.scss`, which we only import in `input-field.scss` and then overwrite the background value.

So I've removed them form here, and it seems to look just fine 🙂

Reply:
🤘